js/classes/entities/foreground/surface_snow.js

```diff
@@ -0,0 +1,55 @@
+class SurfaceSnow {
+    constructor() {
+        this.pos = new Vector(0, 0);
+
+        this.animator = new Animator(Precipitation.SPRITESHEET, new Vector(0, 7 * Precipitation.SIZE.x), new Vector(Precipitation.SIZE.x, Precipitation.SIZE.y), 1, 1);
```

Comment:
Nit: This line is pretty long, 

```suggestion
        this.animator = new Animator(
            Precipitation.SPRITESHEET,
            new Vector(0, 7 * Precipitation.SIZE.x),
            new Vector(Precipitation.SIZE.x, Precipitation.SIZE.y),
            1,
            1,
        );
```